python/pom.xml

```diff
@@ -16,7 +16,7 @@
     <description>Generators to create python for rosetta</description>
     <properties>
         <!-- used by PythonFilesGeneratorTest::generatePython to generate python from a rosetta source --> 
-        <cdm.rosetta.source.path>build/resources/rosetta/cdm/5_8_0</cdm.rosetta.source.path>

```

Comment:
this should be 

<cdm.rosetta.source.path>[Path to Rosetta]</cdm.rosetta.source.path>